sos/report/plugins/__init__.py

```diff
@@ -2332,7 +2332,7 @@ def add_service_status(self, services, **kwargs):
     def add_journal(self, units=None, boot=None, since=None, until=None,
                     lines=None, allfields=False, output=None,
                     timeout=cmd_timeout, identifier=None, catalog=None,
-                    sizelimit=None, pred=None, tags=[]):
+                    sizelimit=None, otherargs=None, pred=None, tags=[]):
```

Comment:
Nack to grouping everything under an umbrella argument. If `add_journal()` needs to be extended to support `journalctl` options, the arguments should match the options being supported.

Additionally, we need to properly handle typing (can this take a list? a string? both?) in `add_journal()`. The option handling must also do the string formatting here, not relying on doing this in various plugin's `setup()` method which would be more error prone.